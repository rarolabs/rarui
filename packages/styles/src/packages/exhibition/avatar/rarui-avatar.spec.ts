import { avatar } from ".";

describe("Avatar Styles", () => {
  it("container class should have the correct styles", () => {
    expect(avatar.classnames.container).toMatchSnapshot();
  });

  describe("avatar recipe", () => {
    it("should apply base styles", () => {
      const className = avatar.classnames.avatar();
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for small size", () => {
      const className = avatar.classnames.avatar({ size: "small" });
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for medium size", () => {
      const className = avatar.classnames.avatar({ size: "medium" });
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for large size", () => {
      const className = avatar.classnames.avatar({ size: "large" });
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for xLarge size", () => {
      const className = avatar.classnames.avatar({ size: "xLarge" });
      expect(className).toMatchSnapshot();
    });
  });

  describe("badge recipe", () => {
    it("should apply correct styles for small size", () => {
      const className = avatar.classnames.badge({ size: "small" });
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for medium size", () => {
      const className = avatar.classnames.badge({ size: "medium" });
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for large size", () => {
      const className = avatar.classnames.badge({ size: "large" });
      expect(className).toMatchSnapshot();
    });

    it("should apply correct styles for xLarge size", () => {
      const className = avatar.classnames.badge({ size: "xLarge" });
      expect(className).toMatchSnapshot();
    });
  });
});
