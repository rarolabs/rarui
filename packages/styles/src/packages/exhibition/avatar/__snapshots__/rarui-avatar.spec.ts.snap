// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Styles avatar recipe should apply base styles 1`] = `"rarui-avatar__1jwfew51"`;

exports[`Avatar Styles avatar recipe should apply correct styles for large size 1`] = `"rarui-avatar__1jwfew51 rarui-avatar_size_large__1jwfew54"`;

exports[`Avatar Styles avatar recipe should apply correct styles for medium size 1`] = `"rarui-avatar__1jwfew51 rarui-avatar_size_medium__1jwfew53"`;

exports[`Avatar Styles avatar recipe should apply correct styles for small size 1`] = `"rarui-avatar__1jwfew51 rarui-avatar_size_small__1jwfew52"`;

exports[`Avatar Styles avatar recipe should apply correct styles for xLarge size 1`] = `"rarui-avatar__1jwfew51 rarui-avatar_size_xLarge__1jwfew55"`;

exports[`Avatar Styles badge recipe should apply correct styles for large size 1`] = `"rarui-avatar_badge__1jwfew56 rarui-avatar_badge_size_large__1jwfew59"`;

exports[`Avatar Styles badge recipe should apply correct styles for medium size 1`] = `"rarui-avatar_badge__1jwfew56 rarui-avatar_badge_size_medium__1jwfew58"`;

exports[`Avatar Styles badge recipe should apply correct styles for small size 1`] = `"rarui-avatar_badge__1jwfew56 rarui-avatar_badge_size_small__1jwfew57"`;

exports[`Avatar Styles badge recipe should apply correct styles for xLarge size 1`] = `"rarui-avatar_badge__1jwfew56 rarui-avatar_badge_size_xLarge__1jwfew5a"`;

exports[`Avatar Styles container class should have the correct styles 1`] = `"rarui-avatar_container__1jwfew50"`;
